Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -13,6 +13,115 @@ let version = "2.0.0"
 
 public class Klaviyo: NSObject  {
     
+    public struct Event {
+        public struct Attributes {
+            public struct Metric {
+                public let name: String
+                public let service: String?
+                public init(name: String,
+                            service: String?="ios-analytics") {
```

Comment:
should we default on line 24 instead? does doing `Metric(name: "some name", service: nil)` bypass the default here?

Reply:
Yah I guess we don't need them to change this.